py_setenv.py

```diff
@@ -107,8 +107,9 @@ def get_variable(name, user):
             value, regtype = winreg.QueryValueEx(key, name)
         return value
     except WindowsError:
-        click.echo("Environment Variable '{}' does not exist".format(name))
-        return ""
+        if not suppress_echo:
+            click.echo("Environment Variable '{}' does not exist".format(name))
+        raise KeyError
```

Comment:
I have some concerns how this will work in CLI implementation. It will be a long unreadable traceback.

I think we need to raise a `KeyError` for Python module and just return empty and error message in CLI

Reply:
also unittests fail with current implementation. Need to fix it as well